less/home-page.less

```diff
@@ -72,4 +145,42 @@
       }
     }
   }
-}// home
\ No newline at end of file
+}// home
+
+// Contact Us styling
+
+form {
+  margin: 0 auto;
+  width: 500px;
+  padding: 1em;
+  border: 1px solid black;
+  border-radius: 1em;
```

Comment:
ems are responsive, but not as responsive as rems. Try to use rems whenever possible at Lambda School. And then do whatever your employer wants when you get hired! :D